src/server.cpp

```diff
@@ -942,7 +959,9 @@ void Server::Receive()
 			}
 
 			peer_id = pkt.getPeerId();
+			m_packet_recv_counter->increment();
 			ProcessData(&pkt);
+			m_packet_recv_processed_counter->increment();
```

Comment:
These will almost always be equal so I don't see this being a very useful metric.
So just `m_packet_recv_counter` would be enough.

Reply:
in fact not, because ProcessData can trigger an exception, having 2 metrics permits to measure the number of broken packet processing. That can be useful to find why some clients are broken, because they sent bad packets